processor/src/operations/decorators.rs

```diff
@@ -120,6 +195,147 @@ mod tests {
         assert_eq!(expected, process.stack.trace_state());
     }
 
+    #[test]
+    fn test_debug() {
+        // Logger can only be initialized once per module, therefore grouping all into one test.
+        let mut logger = Logger::start();
+        test_print_mem(&mut logger);
+        test_print_stack(&mut logger);
+        test_print_local(&mut logger);
+    }
```

Comment:
maybe we should move these tests into a separate file? For example, we can convert `decorators.rs` into a separate directory with `mod.rs` and `debug_tests.rs` files.